packages/ocular-ui/services/admin-api.ts

```diff
@@ -19,9 +19,9 @@ export default {
       const path = `/admin/apps`;
       return ocularRequest('GET', path);
     },
-    retrieve(id) {
-      const path = `/admin/apps/${id}`;
-      return ocularRequest('GET', path);
+    retrieveApp(data) {
```

Comment:
Revert this to the original state. We should never pass any extra data as params here. Check how other requests are passing data to the back end.

```
```

Reply:
For example check

teams.addmembers in this file

Reply:
I am passing only app name in post request , isn't that what we supposed to do in post request, I am not passing any additional data in data object

Reply:
retrieve(id) {
      const path = `/admin/apps/${id}`;
      return ocularRequest('GET', path , data);
    },
    
  where id is the id of the installed application in this case the Oauth.

Reply:
Done